docs/guide/advanced/conditional-fields.md

```diff
@@ -33,6 +33,8 @@ add: {
 }
 ```
 
+## Multiple required conditions
+
 **The `if` setting may contain more than one condition.** When there is more than one, all conditions must be met before the field will be active.
 
 In the next example, `seenMovie` must be `true` *and* `relationship` must be `'none'` for the rating field to appear.
```

Comment:
For purposes of an example in our documentation, `relationship` is a potentially confusing field name.

Reply:
I agree, I'm not sure what would go here. It could be confused with a relationship field (unlikely, but still). Also, what kind of relationships do most people have with filmmakers. By most people, I mean people that aren't stars living in the hills of LA, like myself.

Reply:
I'll try "connection." "association" and "relation" are also options.

I had in my head the case of an audience at a film festival or something like that.